### ISSUE TITLE
Add option to write parquet output

### ISSUE THREAD

**Issue by shntnu** [cytomining/pycytominer#117]
Currently, [`cyto_utils.output`](https://github.com/cytomining/pycytominer/blob/master/pycytominer/cyto_utils/output.py#L12-L62) can write out `csv`, `csv.gz`, and `tsv`.

We'd like it to also be able to write out [`parquet`](https://arrow.apache.org/docs/python/parquet.html). 

The immediate use case we have in mind is converting SQLite backends to Parquet format, which make it a lot easier to work with large single-cell datasets.

This will be kinda redundant with [`cytominer-database`](https://github.com/cytomining/cytominer-database) functionality which can directly write out parquet but it will serve our very frequent need of working with existing large SQLite files.

**Comment by shntnu** [comment]
I wonder if it is as simple as this (needs to be tested)

```py
pyarrow.parquet.write_table(pyarrow.Table.from_pandas(df) , output_filename)
```

**Comment by shntnu** [comment]
I've tested it and that (`pyarrow.parquet.write_table(pyarrow.Table.from_pandas(df), output_filename)`) is probably all we need to do here

https://colab.research.google.com/drive/1D0J7FiqmzR49IjSKy2lQVtZ8Gez4cDkD?authuser=1#scrollTo=W3FJmSQXgBOd

**Comment by gwaygenomics** [comment]
related to #112

**Comment by gwaybio** [comment]
addressed in 2327ebccc867b8f9514b8b0edd42fb93ceedbf1e
